doc/minetestserver.6

```diff
@@ -1,8 +1,7 @@
-.\" Minetestserver man page
 .TH minetestserver 6 "10 September 2013" "" ""
 
 .SH NAME
-minetestserver \- Minetest server
+minetestserver \- Minetest server-only binary
```

Comment:
IMO `Minetest server` is clear enough.
Also saying something is a binary misses the point of the description.